unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Utils/ComponentUtils.cs

```diff
@@ -0,0 +1,94 @@
+﻿using DCL.Components;
+using DCL.Models;
+using UnityEngine;
+
+namespace DCL.Controllers
+{
+    public static class ComponentUtils
```

Comment:
Can we add some early returns on all the methods, I find the nesting more difficult to read

Reply:
Done